doc/reference/reference_lua/box_space/create_index.rst

```diff
@@ -8,10 +8,10 @@ On this page:
 * :ref:`create_index() description <box_space-create_index>`
 * :ref:`Details about index field types <details_about_index_field_types>`
 * :ref:`Allowing null for an indexed key <box_space-is_nullable>`
-* :ref:`Using field names instead of field numbers <box_space-field_names>`
-* :ref:`Using the path option for map fields (JSON-indexes) <box_space-path>`
-* :ref:`Using the path option with [*] <box_space-path_multikey>`
-* :ref:`Making a functional index with space_object:create_index() <box_space-index_func>`
+* :ref:`Creating an index using field names instead of field numbers <box_space-field_names>`
+* :ref:`Creating an index using the path option for map fields (JSON-path indexes) <box_space-path>`
+* :ref:`Creating an index using the path option with [*] <box_space-path_multikey>`
```

Comment:
Maybe we should call `[*]` wildcard?

It's only suggestion and should be discussed.

Reply:
There are several wildcard characters. https://support.microsoft.com/en-us/office/examples-of-wildcard-characters-939e153f-bd30-47e4-a763-61897c87b3f4

This one is called an asterisk.